src/navigator-context-menu.jsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
```

Comment:
Pls update the date and add a comment that this is direct copy from cockpit repo with a link.